hivemq-edge/src/main/java/com/hivemq/api/resources/impl/ProtocolAdaptersResourceImpl.java

```diff
@@ -380,20 +396,20 @@ public int getDepth() {
                     log.trace("Adapter '{}' was started successfully.", adapterId);
                 }
             });
-            case STOP -> protocolAdapterManager.stopAsync(adapterId, false).whenComplete((result, throwable) -> {
+            case STOP -> protocolAdapterManager.stopAsync(adapterId).whenComplete((result, throwable) -> {
                 if (throwable != null) {
                     log.error("Failed to stop adapter '{}'.", adapterId, throwable);
                 } else {
                     log.trace("Adapter '{}' was stopped successfully.", adapterId);
                 }
             });
-            case RESTART -> protocolAdapterManager.stopAsync(adapterId, false)
-                    .thenRun(() -> protocolAdapterManager.startAsync(adapterId))
+            case RESTART -> protocolAdapterManager.stopAsync(adapterId)
+                    .thenCompose(ignored -> protocolAdapterManager.startAsync(adapterId))
                     .whenComplete((result, throwable) -> {
                         if (throwable != null) {
                             log.error("Failed to restart adapter '{}'.", adapterId, throwable);
                         } else {
-                            log.trace("Adapter '{}' was restarted successfully.", adapterId);
+                            log.info("Adapter '{}' was restarted successfully.", adapterId);
```

Comment:
Should it be `trace`?

Reply:
That actually has an information character.
I would suggest keeping it at info.
These operations are very low frequence.

Reply:
The reason I asked this question is the other similar logging statements are all `trace` ones, but this one is `info`.